src/app/components/tfrs/tfrs.component.scss

```diff
@@ -0,0 +1,39 @@
+table {
```

Comment:
Do not use generic css you are using table selector directly and will give same style to all the tables inside component , for example in future someone added one more table then it will also take same style as table selector is styled